recipes/RavenPy/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "ravenpy" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 35181f3a4a4b42b9a8d98e78df8b5df1d6e4ae8fd8310a9826c93d88efe2a312
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - setuptools
+    - numpy >=1.20
+    - gdal >=3
+    - pip
+  run:
+    - raven-hydro ==3.0.4.322
+    - ostrich ==21.03.16
+    - python
+    - affine
+    - click
+    - climpred >=2.1
+    - clisops >=0.6.3
+    - dask <=2021.02.0
+    - gdal >=3
+    - geopandas >=0.9
+    - libnetcdf
+    - lxml
+    - matplotlib
```

Comment:
Unless you need a PyQt GUI, best to depend on `matplotlib-base`.

Reply:
Done.